docs/Project.toml

```diff
@@ -1,7 +1,11 @@
 [deps]
+Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 GR = "28b8d3ca-fb5f-59d9-8090-bfdbd6d07a71"
 
 [compat]
 Documenter = "1"
```

Comment:
We might want to proactively specify a compat entry here if we add Distributions to the explicit dependencies (which IMO is good). Otherwise CompatHelper will open a PR:

```suggestion
Distributions = "0.25"
Documenter = "1"
```